src/olympia/addons/models.py

```diff
@@ -1323,6 +1324,28 @@ def promoted_group(self, *, group=None, application=None,
             not currently_approved or promoted.is_addon_currently_promoted)
         return promoted.group if is_promoted else NOT_PROMOTED
 
+    def promoted_applications(self):
+        """Returns a list of applications for which the addon is currently
+        promoted, or None if the addon is not currently promoted.
+        """
+        promoted_group = self.promoted_group()
+        if promoted_group != NOT_PROMOTED:
```

Comment:
you can just do `if promoted_group` - `NOT_PROMOTED` is falsey.  (Or just `if self.promoted_group()`)